spec/components/layout_for_public_spec.rb

```diff
@@ -59,6 +59,13 @@ def component_name
     assert_select ".gem-c-layout-for-public .gem-c-feedback", false
   end
 
+  it "can omit the footer navigation" do
+    render_component(omit_footer_navigation: true)
+
+    assert_select ".gem-c-layout-for-public .govuk-footer__navigation", false
+    assert_select ".gem-c-layout-for-public .govuk-footer__section-break", false
+  end
```

Comment:
Is it worth checking that it won't omit the footer navigation if `omit_footer_navigation` is `false`?

Reply:
Updated